apps/client/src/features/cuesheet/cuesheet-progress/CuesheetProgress.tsx

```diff
@@ -14,9 +14,9 @@ export default function CuesheetProgress() {
       now={timer.current}
       complete={totalTime}
       normalColor={data!.normalColor}
-      warning={data!.warningThreshold}
+      warning={timer.timeWarning || 0}
```

Comment:
would it not be best if the progressbar handled undefined?

Reply:
i agree